README.md

```diff
@@ -148,6 +148,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   ### JavaScript
   - :file_folder: [JavaScript official documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
   - :video_camera: [Full course](https://www.youtube.com/watch?v=lI1ae4REbFM)
+  - :video_camera: [Project Based course](https://www.youtube.com/watch?v=lI1ae4REbFM)
```

Comment:
hey, this course is already added in line `150` do one thing, delete the above link in line `150` and then make PR